servers/mcp-neo4j-cloud-aura-api/Makefile

```diff
@@ -0,0 +1,16 @@
+install-dev:
```

Comment:
could we move these to uv and pyproject? (not crucial just for consistency later)